ow_faults/src/FaultInjector.cpp

```diff
@@ -211,12 +221,22 @@ void FaultInjector::jointStateCb(const sensor_msgs::JointStateConstPtr& msg)
     setPowerTemperatureFaultValue(false);
   }
 
+  if (m_faults.antennae_pan_failure){
+    systemFaultsBitmask |= isPanTiltExecutionError;
+    setPTFaultsMessage(pt_faults_msg,hardwareFault);
+  }
+  if (m_faults.antennae_tilt_failure) {
+    systemFaultsBitmask |= isPanTiltExecutionError;
+    setPTFaultsMessage(pt_faults_msg,hardwareFault);
+  }
+
```

Comment:
Refactoring suggestion to reduce the code duplication
```suggestion
  if (m_faults.antennae_pan_failure || m_faults.antennae_tilt_failure) {
    systemFaultsBitmask |= isPanTiltExecutionError;
    setPTFaultsMessage(pt_faults_msg,hardwareFault);
  }
```